clang-tools-extra/docs/clang-tidy/checks/misc/coroutine-hostile-raii.rst

```diff
@@ -0,0 +1,48 @@
+.. title:: clang-tidy - misc-coroutine-hostile-raii
+
+misc-coroutine-hostile-raii
+====================
+
+This check detects hostile-RAII objects which should not persist across a 
```

Comment:
first paragraph in documentation should be same as on in release notes and as in .hpp file for a class description